.github/workflows/build.yml

```diff
@@ -1,26 +1,59 @@
 name: 'build'
 
-on: [push, pull_request]
+on:
+  pull_request:
+  push:
+    branches:
+      - main
```

Comment:
Is there any reason to limit the push only to the main branch?

Reply:
Yes, previously a push to the feature branch was triggering the action on both "pull requests" and "push", making every jobexecute twice.